ci-operator/step-registry/stolostron-engine/ci/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
These symlinks aren't necessary. The owners files will cascade to child directories.

Reply:
Ok.  Removed.